file-formats/fugr/type/zif_fb_aff_fugr_v1.intf.abap

```diff
@@ -0,0 +1,53 @@
+INTERFACE zif_fb_aff_fugr_v1
+  PUBLIC.
+
+  "! <p class="shorttext">Status</p>
+  "! status
+  "! $values {@link zif_fb_aff_fugr_v1.data:co_status}
+  TYPES ty_status TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Status</p>
+    "! status
+    BEGIN OF co_status,
+      "! <p class="shorttext">SAP Program</p>
+      "! SAP program
+      sap_program      TYPE ty_status VALUE 'P',
+      "! <p class="shorttext">Customer Program</p>
+      "! Customer Production Program
+      customer_program TYPE ty_status VALUE 'K',
+      "! <p class="shorttext">System Program</p>
+      "! System program
+      system_program   TYPE ty_status VALUE 'S',
+      "! <p class="shorttext">Test Program</p>
+      "! Test program
+      test_program     TYPE ty_status VALUE 'T',
+      "! <p class="shorttext">Not Classified</p>
+      "! Not classified
+      not_classified   TYPE ty_status VALUE space,
+    END OF co_status.
+
+
+  TYPES:
+    "! <p class="shorttext">Object Type FUGR</p>
+    "! FUGR object type
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Schema</p>
+      "! Format version
+      "! $required
+      schema               TYPE string,
+      "! <p class="shorttext">Header</p>
+      "! header
+      "! $required
+      header               TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Fix Point Arithmetic</p>
+      "! Fix point arithmetic
+      "! $required
+      fix_point_arithmetic TYPE abap_bool,
+      "! <p class="shorttext">Status</p>
+      "! status
+      "! $required
+      status               TYPE ty_status,
```

Comment:
I am not sure, whether this is required. I don't think so.

Reply:
I deleted the $required statement